backend/python/app/tests/services/test_story_service.py

```diff
@@ -67,8 +73,60 @@ def test_get_stories_available_for_translation():
     pass
 
 
-def test_create_translation():
-    pass
+def test_create_translation(app, db, services):
+    alphanumeric = "abcdefghijklmnopqrstuvwxyz0123456789"
+
+    translator_authId = "".join(random.choice(alphanumeric) for i in range(10))
```

Comment:
imo i dont think we have to go through the trouble of making randomly generated ids but I also think it's fine that the code for making random ids is here